setup/base.go

```diff
@@ -311,6 +312,17 @@ func (b *BaseDendrite) SetupAndServeHTTP(
 		}
 	}
 
+	externalRouter.Handle("/health",
```

Comment:
Where is best to add a test? I don't see any tests for `/_matrix/client/versions` for example.